lib/member_records.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+require 'scraped'
+require_relative 'member_record'
+
+class MemberRecords < Scraped::JSON
+  field :members do
+    all_members
+  end
+
+  field :members_of_the_lower_house do
+    lower_house_members.map do |mem|
+      fragment mem => MemberRecord
+    end
+  end
+
+  field :next do
+    json[:next]
+  end
+
+  private
+
+  def all_members
```

Comment:
I've renamed the class to `MemberRecordsPage` to make it clear that it represents a single page.